packages/get-safe-property-chain/package.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
They point number 4 in the bullet list is not done. 
![image](https://user-images.githubusercontent.com/4663897/122215050-157a5400-ceab-11eb-80f0-6689cca62e6e.png)